docker_prod/docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+services:
+  app:
+    image: archivesspace/archivesspace:4.0.0-RC1
+    container_name: archivesspace
+    restart: on-failure
+    ports:
+      - "8080:8080"
+      - "8081:8081"
+      - "8082:8082"
+      - "8089:8089"
+      - "8090:8090"
+    depends_on:
+      - db
+      - solr
+    environment:
+      APPCONFIG_DB_URL: "jdbc:mysql://db:3306/archivesspace?useUnicode=true&characterEncoding=UTF-8&user=as&password=as123&useSSL=false&allowPublicKeyRetrieval=true"
+      APPCONFIG_FRONTEND_PROXY_URL: "http://localhost:8080"
+      APPCONFIG_PUBLIC_PROXY_URL: "http://localhost:8081"
+      APPCONFIG_SOLR_URL: "http://solr:8983/solr/archivesspace"
+      ASPACE_DB_MIGRATE: true
+      ASPACE_JAVA_XMX: "-Xmx2048m"
+      JAVA_OPTS: "-Djava.awt.headless=true -Dfile.encoding=UTF-8 -server -Xss1024k -Djavax.accessibility.assistive_technologies=''"
+      DB_ADDR: db
+      DB_NAME: archivesspace
+      DB_USER: as
+      DB_PASS: as123
+      SOLR_URL: http://solr:8983/solr/archivesspace
```

Comment:
These are redundant / not actually used (The `APPCONFIG_` values are the real ones).